ios/Mattermost/AppDelegate.m

```diff
@@ -71,24 +71,18 @@ -(void)application:(UIApplication *)application didReceiveRemoteNotification:(no
   NSString* action = [userInfo objectForKey:@"type"];
   NSString* channelId = [userInfo objectForKey:@"channel_id"];
   NSString* ackId = [userInfo objectForKey:@"ack_id"];
+  RuntimeUtils *utils = [[RuntimeUtils alloc] init];
 
-  if (action && [action isEqualToString: NOTIFICATION_CLEAR_ACTION]) {
```

Comment:
Not well-versed with this codebase, so perhaps a silly question: is this refactoring related to this PR or just a general thing ?

Reply:
General, while looking at this I saw the there was unnecessary duplicate code